### PR TITLE
Improve code snippets for setup with Ruby on Rails

### PR DESCRIPTION
**Pull request by ledermann**
The response example now matches the PHP one.

### CODE REVIEW

**Comment by reinink** [comment]
Ping @bknoles and @BrandonShar. Can you please review this before I merge it in?

File: pages/server-side-setup.mdx [hunk 1]
```diff
@@ -135,10 +135,14 @@ That's it, you're all ready to go server-side! From here you can start creating
       language: 'ruby',
       code: dedent`
         class EventsController < ApplicationController
-          def index
-            render inertia: 'Events',
+          def show
+            @event = Event.find(params[:id])
```
Comment (BrandonShar) [inline pages/server-side-setup.mdx]: Same comment as #33 to prefer a local variable over an instance variable here.

Comment (ledermann) [reply pages/server-side-setup.mdx]: Ok, fixed.

**Comment by reinink** [comment]
Thanks gents! 👏

**Pull request merged**